packages/okta-auth-js/test/spec/tokenManager.js

```diff
@@ -20,7 +20,8 @@ function setupSync(options) {
     redirectUri: 'https://example.com/redirect',
     tokenManager: {
       expireEarlySeconds: options.tokenManager.expireEarlySeconds || 0,
-      storage: options.tokenManager.type,
+      storage: options.tokenManager.storage,
```

Comment:
💯  to this change - having the tests match the naming is one of those subtle things that pays off so much down the line.